src/order/boolean_algebra.lean

```diff
@@ -768,6 +768,9 @@ boolean_algebra.of_core
   top_le_sup_compl := λ _, inf_compl_eq_bot.ge,
   ..order_dual.distrib_lattice α, ..order_dual.bounded_order α }
 
+@[simp] lemma of_dual_compl (a) : of_dual aᶜ = (of_dual a : α)ᶜ := rfl
+@[simp] lemma to_dual_compl (a) : to_dual (aᶜ : α) = (to_dual a)ᶜ := rfl
```

Comment:
Can you make the types of the arguments explicit here?

Also, can you add the lemmas about `sdiff` too?

Reply:
No I can't add the lemma about `sdiff`, because its dual is Heyting implication, which we don't have yet. I have it ready on a branch, however.